doc/architecture/decisions/0006-backend-is-deployed-on-more-than-one-instance.md

```diff
@@ -21,7 +21,11 @@ We decided to use the Premium version that provides us the following features:
 * Scale out to multiple cache units
 * Azure Virtual Network
 
+From the external the cluster appears as a single connection so there is no need to use a Redis client that supports
```

Comment:
I've read that it needs a client library that supports clustering, where did you get this info ?

Reply:
you are right, documentation is confusing, [here](https://docs.microsoft.com/en-us/azure/redis-cache/cache-how-to-premium-clustering) it says that "At the present time not all clients support Redis clustering", but after that it says "You can connect to your cache using the same endpoints, ports, and keys that you use when connecting to a cache that does not have clustering enabled. Redis manages the clustering on the backend so you don't have to manage it from your client.". Anyway the nodejs example [here](https://docs.microsoft.com/en-us/azure/redis-cache/cache-nodejs-get-started) uses the same node_redis package as we do.